### ISSUE TITLE
After installation, nothing occurs

### ISSUE THREAD

**Issue by nacesprin** [fonfonx/atom-unsaved-changes#2]
Hello. After the package installation and to able it, I change multiple lines, but nothing occurs.
What may be ocurring?

**Comment by fonfonx** [comment]
How did you install the package? Did you reload Atom (`Ctrl + Alt + R`)?  In the "Packages" tab do you see "Atom Unsaved changes"?

**Comment by nacesprin** [comment]
Yes, I achieve it, thanks. But to see the changes, do I need to reload every time the "Packages tab\Atom Unsaved Changes". Is there any way to refresh automatically the changes?

And could it be possible to go at changed line when click in the unsaved line?

Thanks!!

**Comment by fonfonx** [comment]
Currently you have to press again `Ctrl + Alt + U` to see the new changes.

You mean, when you click on the green line, go to the corresponding line in your editor? If I have some time to work on the package I may look at this feature.

**Comment by nacesprin** [comment]
Hello @fonfox . I think to mark the line in the editor is a good valued solution, like this (appart of the pannel "atom unsaved changes" that already exists, of course):

https://forum.sublimetext.com/t/hightlight-changed-row/3187/10

Is there any way to achieve that?

This is no a particular need for me, but I think it would be very helpful that this plugin works like the mentioned on sublimetext for all programmers.
